README.md

```diff
@@ -11,17 +9,16 @@ post.description = "Desc goes here..."
 post.save
 ```
 
-This syntax will let you manually create a new `Post` record with a `title` and `description` attribute. After running the `save` method in the console you will see output similar to the following:
+This syntax will let you manually create a new `Post` record with `title` and `description` attributes. After running the `save` method in the console, you will see output similar to the following:
 
 ```shell
  (0.1ms)  begin transaction
 SQL (0.3ms)  INSERT INTO "posts" ("title", "description", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["title", "Title Goes Here"], ["description", "Desc goes here..."], ["created_at", "2015-11-23 22:26:43.799742"], ["updated_at", "2015-11-23 22:26:43.799742"]]
  (1.2ms)  commit transaction
 => true
-
 ```
 
-As you can see, the `save` method generates a SQL script that inserts a new record into the database, passing the `title` and `description` parameters into the statement and then returning the newly created `Post` object. At a high level, this is what the `create` method in our controller will be doing.
+As you can see, the `save` method generates a SQL script that inserts a new record into the database, passing each of the `Post` object's attributes into the statement. At a high level, this is what the `create` method in our `PostsController` will be doing.
```

Comment:
Perhaps we should fill the void of the return the newly created post object statement with '...and then returning `true` indicating that the object was saved successfully' or something of the sort?